Add a task info section when there is a new ocis release (#602)
owncloud/docs-ocis: docs/new-infinite-scale-release-info.md

> # New Infinite Scale Releas Info

Reviewer: ```suggestion
# New Infinite Scale Release Info
```

> If there is a new Infinite Scale release on the way, there are some things to know respectively to check:

Reviewer: ```suggestion
If there is a new Infinite Scale release on the way, there are some things to check:
```

**Relevant for docs:**

> * The docs relevant releasing process in the `ocis` repo works the following:

Reviewer: ```suggestion
* The docs relevant releasing process in the `ocis` repo works the following way:
```

>   * Based on a release tag like `v4.0.0`, a new `stable-x` branch is created.

Reviewer: I think that we create a branch like `stable-4.0` - there is no `v` in the branch name. There is no patch version in the branch name. If what I said is correct, then detail can be added here:
```suggestion
  * Based on a release tag like `v4.0.0`, a new `stable-x` branch is created. For example, `stable-4.0`.
```

>   * A new and empty `docs-stable-x` branch is created.

Reviewer: ```suggestion
  * A new and empty `docs-stable-x` branch is created. For example, `docs-stable-4.0`.
```

  * The pipeline of the `stable-x` branch gets adapted to write to the `docs-stable-x` branch.
  * The `stable-x` branch gets added to the nightly cron jobs in drone which populates new and changed content to `docs-stable-x`.
  * Any merge in the `stable-x` branch also triggers the pipeline as usual.

> * **IMPORTANT** For any changes necessary that effect the output of the `docs-stable-x`, these must be done in `stable-x` and take effect in `docs-stable-x` when the `stable-x` pipeline ran. Manual commits to `docs-stable-x` will get overwritten at least daily.

Reviewer: ```suggestion
* **IMPORTANT** For any changes necessary that effect the output of the `docs-stable-x` branch, these must be done in `stable-x` and take effect in `docs-stable-x` when the `stable-x` pipeline runs. Manual commits to `docs-stable-x` will get overwritten at least daily.
```

> * **IMPORTANT**: Any changes neccessary in `stable-x` are based on a 6-eye principle and need 2 approvals therefore.

Reviewer: ```suggestion
* **IMPORTANT**: Any changes neccessary in `stable-x` are based on a 6-eye principle and therefore need 2 approvals.
```
